x-pack/plugins/elastic_assistant/server/schemas/capabilities/get_capabilities_route.gen.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { z } from 'zod';
+
+/*
+ * NOTICE: Do not edit this file manually.
+ * This file is automatically generated by the OpenAPI Generator, @kbn/openapi-generator.
```

Comment:
If the `@kbn/openapi-generator` (that generated this file) is enhanced (in a future PR) to optionally accept something like `typesOutputDir`, consider updating the `@kbn/openapi-generator`, such that generated comments like this one include a token and / or the `info` metadata from the source schema:

```yaml
info:
  title: Get Capabilities API endpoint
  version: '1'
```

Including a token and / or the `info` metadata from the source schema would make it possible via search / `grep` to locate the source schema that defines the types. This would be useful if, for example, `typesOutputDir` was configured to be a path in the `kbn-elastic-assistant-common` package.

Reply:
Yes yes, great feedback! In the follow-up PR I'll update the generated comments with more information like this and back-references to the source file 👍

Reply:
Draft PR with fix: https://github.com/elastic/kibana/pull/174718. Will still access if we just want to move schema files to `common`, but this is what the fix would be if we go this route.